docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,28 @@ kotlin {
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Sharing Kotlin cinterop between dependent Pods
+
+If you're adding multiple dependencies on Pods using the `pod()` function, you might encounter issues in
+case there are dependencies between APIs of your Pods.
+
+To make the code compile in this case, use the `useInteropBindingFrom()` function.
+It utilizes the cinterop binding generated for another Pod when building a binding for the new Pod.
+
+The dependent Pod should be declared before setting up the dependency:
+
+```kotlin
+// The cinterop of pod("WebImage"):
+fun loadImage(): WebImage
+
+// The cinterop of pod("Info"):
+fun printImageInfo(image: WebImage)
+
+// Your code:
+printImageInfo(loadImage())
+```
+
+If you didn't set up the correct dependencies between cinterops in this case,
+the code would be invalid because the `WebImage` type would come from different cinterops and, thus, packages.
```

Comment:
...would be sourced from different cinterop files and, consequently, different packages.